src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
ok ok, got it!
@darkbasic do you think that you will be able to add that support to `angular-meteor-docs`?  if not we'll have to wait for @dotansimha